crates/blockchain-tree/src/blockchain_tree.rs

```diff
@@ -724,24 +665,10 @@ impl<DB: Database, EF: ExecutorFactory> BlockchainTree<DB, EF> {
         Ok(())
     }
 
-    /// Check if block is found inside chain and if the chain extends the canonical chain.
-    ///
-    /// if it does extend the canonical chain, return `BlockStatus::Valid`
-    /// if it does not extend the canonical chain, return `BlockStatus::Accepted`
+    /// Returns `true` if block is found inside the tree.
     #[track_caller]
```

Comment:
can probably remove `track_caller` here because this function shouldn't panic any more